programs/Makefile

```diff
@@ -52,13 +52,19 @@ DEBUGFLAGS= -Wall -Wextra -Wundef -Wcast-qual -Wcast-align -Wshadow \
             -Wswitch-enum -Wdeclaration-after-statement -Wstrict-prototypes \
             -Wpointer-arith -Wstrict-aliasing=1
 CFLAGS   += $(DEBUGFLAGS) $(MOREFLAGS)
+
+include ../Makefile.inc
+
+ifeq ($(TARGET_OS)$(shell uname -r),SunOS5.10)
```

Comment:
What's the purpose of listing both `TARGET_OS` and `$(shell uname -r)`

since `TARGET_OS` is already defined as `TARGET_OS ?= $(shell uname)` ?

Reply:
lz4 builds fine on Solaris 11 without it so I thought it was unnecessary to add it to all Solaris versions. I don't have access to any < 10 machines for testing either.